HLTrigger/Configuration/python/HLT_75e33/modules/hgcalLayerClustersL1Seeded_cfi.py

```diff
@@ -1,16 +1,24 @@
 import FWCore.ParameterSet.Config as cms
 
-hgcalLayerClustersL1Seeded = cms.EDProducer("HGCalLayerClusterProducer",
-    HFNoseInput = cms.InputTag("HGCalRecHitL1Seeded","HGCHFNoseRecHits"),
-    HGCBHInput = cms.InputTag("hltRechitInRegionsHGCAL","HGCHEBRecHits"),
-    HGCEEInput = cms.InputTag("hltRechitInRegionsHGCAL","HGCEERecHits"),
-    HGCFHInput = cms.InputTag("hltRechitInRegionsHGCAL","HGCHEFRecHits"),
-    detector = cms.string('all'),
-    doSharing = cms.bool(False),
-    mightGet = cms.optional.untracked.vstring,
-    nHitsTime = cms.uint32(3),
-    plugin = cms.PSet(
-        dEdXweights = cms.vdouble(
+hgcalLayerClustersEEL1Seeded = cms.EDProducer('HGCalLayerClusterProducer',
```

Comment:
I think the idea is that every `cfi` should contain only a single module, named like the module it defines, e.g. `hgcalLayerClustersEEL1Seeded_cfi.py`, etc.